nanoFramework.Device.Bluetooth/BluetoothEventListener.cs

```diff
@@ -258,6 +304,50 @@ private GattLocalCharacteristic FindCharacteristic(ushort id)
             return null;
         }
 
+        #endregion
+
+        #region Session routing
+        public BluetoothLEServer BluetoothServer { get => _server; set => _server = value; }
+
+        //public void AddSecurity(Pairing s)
```

Comment:
Why are you commenting all this sections? If you want to keep this dead code, please add a comment otherwise, please remove so we can keep things clean.

Reply:
Thought I had scanned all files for commented out code. Obviously missed this one.